apps/website/docs/magazine/migration_from_redux.md

```diff
@@ -46,7 +46,36 @@ export const reduxInterop = createReduxIntegration({
 });
 ```
 
-☝️ Notice, how explicit `setup` event is required to initialize the interoperability. Usually it would be an `appStarted` event or any other "app's lifecycle" event.
+#### Avoiding dependency cycles
+
+Note, that overload of the `createReduxIntegration` with explicit `reduxStore` allows for better Typescript type inference, but also might result in cyclic dependencies.
```

Comment:
It seems like a non-default case for me. Shall we hide it under `::: details`?

Reply:
So far, all the real users of the library i know about, had this exact issue 😅 

While there is only a few real-life users, i feel like, this should not be hidden from the reader, so one will get a heads-up about this problem and the solution for it right away.

Time will tell - this may be the behavior that should become the default